src/frontend/Komponenter/Behandling/Totrinnskontroll/SendTilBeslutterFooter.tsx

```diff
@@ -83,13 +133,26 @@ const SendTilBeslutterFooter: React.FC<{
         <>
             {behandlingErRedigerbar && (
                 <Footer>
+                    {kanOppretteFremleggsoppgave && (
+                        <Alertstripe variant={'info'} fullWidth={true}>
+                            Det blir automatisk opprettet oppgave for kontroll av
+                            inntektsopplysninger 1 år frem i tid når denne behandlingen blir
+                            godkjent av beslutter{' '}
+                            <Checkbox
+                                onChange={checkBoxEndring}
+                                checked={skalOppretteFremleggsoppgave}
+                            >
+                                {' '}
+                            </Checkbox>
+                        </Alertstripe>
+                    )}
```

Comment:
Skal denne alerten være her? Ut ifra skissene virker det som at det enten skal vises en alert med kun info dersom `skalOppretteFremleggsoppgave` er true (som gjøres lenger nede), og at denne skal byttes ut med en komponent med valg om hvilke oppgaver som kan ferdigstilles og avhukningsvalg for om det skal opprettes en fremleggsoppgave.

PS: Har bare sett kjapt på skissene så mulig jeg tolker de feil😅

<img width="752" alt="image" src="https://user-images.githubusercontent.com/46678893/226873404-e4b18463-6b1d-4e82-994c-a481de95b9d1.png">

Reply:
Ja, den skissen er egentlig tenkt for hvordan det kanskje kan se ut etterhvert, men nå har vi en enkel sjekkboks med infotekst for denne ene oppgaven

Reply:
Oppdaterte beskrivelsen med nye infoalerts ☝️